SLAM/include/DSOcollector.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
```

Comment:
```c++
#ifndef _header_name_
#define _header_name_
// code
#endif
```
effectively does the same.
I prefer `#pragma once` because it's cleaner. But the above is auto generated by CLion. And so, a lot of my files have these. If you'd prefer, we can make `#pragma once` the standard and I'll slowly convert over. I'm okay with either.